Documentation/teaching/labs/arm_kernel_development.rst

```diff
@@ -266,122 +268,132 @@ Exercises
-   - ``arch/arm/boot/dts/imx6ul-14x14-evk.dtb``, device tree blob for ``i.MX6UL`` board
+* ``arch/arm/boot/Image``, kernel image compiled for ARM
+* ``arch/arm/boot/dts/imx6ul-14x14-evk.dtb``, device tree blob for ``i.MX6UL`` board
 
-Review `Rootfs`_ section and download ``core-image-minimal-qemuarm.ext4`` rootfs.
+Review `rootfs`_ section and download ``core-image-minimal-qemuarm.ext4`` rootfs.
 Run ``qemu`` using then following command:
 
 .. code-block:: bash
 
    ../qemu/build/arm-softmmu/qemu-system-arm -M mcimx6ul-evk -cpu cortex-a7 -m 512M \
-     -kernel arch/arm/boot/zImage -nographic  -dtb arch/arm/boot/dts/imx6ul-14x14-evk.dtb \
+     -kernel arch/arm/boot/Image -nographic  -dtb arch/arm/boot/dts/imx6ul-14x14-evk.dtb \
      -append "root=/dev/mmcblk0 rw console=ttymxc0 loglevel=8 earlycon printk" -sd tools/labs/core-image-minimal-qemuarm.ext4
 
-.. note:: LCDIF and ASRC devices are not well supported with ``Qemu``. Remove them from compilation.
+Once the kernel is booted check the kernel version and CPU information:
 
 .. code-block:: bash
 
-   $ ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- make menuconfig
-   # set FSL_ASRC=n and DRM_MXSFB=n
-   $ ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- make -j8
+    $ cat /proc/version
+    $ cat /proc/cpuinfo
 
-Once the kernel is booted check kernel version and cpu info:
-
-.. code-block:: bash
-
-   $ cat /proc/cpuinfo
-   $ cat /proc/version
-
-2. CPU information
+2. CPU Information
 ------------------
 
-Inspect the CPU configuration for ``NXP i.MX6UL`` board. Start with ``arch/arm/boot/dts/imx6ul-14x14-evk.dts``.
+Inspect the CPU configuration for the NXP i.MX6UL board. Start with ``arch/arm/boot/dts/imx6ul-14x14-evk.dts`` and go through included files.
 
-  - find ``cpu@0`` device tree node and look for ``operating-points`` property.
-  - read the maximum and minimum operating frequency the processor can run
+* Find ``cpu@0`` device tree node and look for the ``operating-points`` property.
+* Read the maximum and minimum operating frequency the processor can run:
 
-   .. code:: bash
+  .. code:: bash
 
       $ cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_min_freq
       $ cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq
 
-3. I/O memory
+3. I/O Memory
 -------------
-Inspect I/O space configuration for ``NXP i.MX6UL`` board. Start with ``arch/arm/boot/dts/imx6ul-14x14-evk.dts`` and identify each device mentioned below.
+
+Inspect I/O space configuration for the NXP i.MX6UL board. Start with ``arch/arm/boot/dts/imx6ul-14x14-evk.dts``, go through included files, and identify each device mentioned below.
 
 .. code:: bash
 
-   $ cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_min_freq
-     00900000-0091ffff : 900000.sram sram@900000
-     0209c000-0209ffff : 209c000.gpio gpio@209c000
-     021a0000-021a3fff : 21a0000.i2c i2c@21a0000
-     80000000-9fffffff : System RAM
+    $ cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_min_freq
```

Comment:
That's not the file that we run cat on. It's `/proc/iomem`

Reply:
Yes. @razvand can you fix this. Other than that it looks good to me.